core/clingutils/src/TClingUtils.cxx

```diff
@@ -3383,6 +3383,7 @@ void ROOT::TMetaUtils::GetFullyQualifiedTypeName(std::string &typenamestr,
                                                  const clang::QualType &qtype,
                                                  const cling::Interpreter &interpreter)
 {
+   cling::Interpreter::PushTransactionRAII RAII(const_cast<cling::Interpreter*>(&interpreter));
```

Comment:
It would be nice to have a motivation for this in the commit log, same for more occurrences below.